src/lib/Components/Bidding/Screens/BidResult.tsx

```diff
@@ -60,35 +67,18 @@ const TopOffset = styled.View`
   align-items: center;
 `
 
-const CenteringContainer = styled.View`
-  flex: 1;
-  justify-content: center;
-  align-items: center;
-`
-
 const Icon = styled.Image`
   width: 20px;
   height: 20px;
   margin: 10px;
 `
 
-const StyledText = styled.Text`
-  margin: 10px;
-  font-family: ${Fonts.GaramondRegular};
-  font-size: 16px;
-  text-align: center;
-  color: #666666;
-`
+const CenteringContainer = props => <Flex flex={1} justifyContent="center" alignItems="center" {...props} />
+
+const StyledText = props => <Serif16 m={3} textAlign="center" color="black60" {...props} />
 
 const TimeLeft = styled.Text`
   font-size: 12px;
   color: #666666;
   margin-top: 10px;
 `
```

Comment:
This `<TimeLeft>` component could also be conslidated. I was just lazy and didn't make it in this PR.